cli/jobs/spark-standalone-jobs/spark_job_word_count_hobo.yml

```diff
@@ -0,0 +1,28 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
```

Comment:
run  /cli/readme.py  to refresh /cli/README.md

Reply:
looks like encountering this issue 

windows is not supported, see issue #146 (https://github.com/Azure/azureml-examples/issues/146)

How did you generate README.md on windows?

Reply:
done